mods/auth/src/utils/jwt.ts

```diff
@@ -17,7 +17,7 @@ export default class JWT implements ITokenManager {
   async encode (
     payload: JwtPayload,
     privateKey: string,
-    expiration: Number = 500
+    expiration: Number = 3000
   ): Promise<string> {
     if (!privateKey) throw new Error('Token generation failure')
```

Comment:
You don't know the reason why the token was not added. This message could be misleading. Instead, you could say "missing required parameter privateKey"